rustic-babel.el

```diff
@@ -73,9 +73,18 @@ should be wrapped in which case we will disable rustfmt."
           (toolchain (cond ((eq toolchain-kw-or-string 'nightly) "+nightly")
                            ((eq toolchain-kw-or-string 'beta) "+beta")
                            ((eq toolchain-kw-or-string 'stable) "+stable")
+                           ((or (null rustic-babel-default-toolchain)
+                                (null (string-blank-p rustic-babel-default-toolchain)))
+                            nil)
                            (toolchain-kw-or-string (format "+%s" toolchain-kw-or-string))
                            (t (format "+%s" rustic-babel-default-toolchain))))
-          (params (list "cargo" toolchain "build" "--quiet"))
+          (params (remove nil (list "cargo"
+                                    (unless (or (null toolchain)
+                                                (= 0 (string-blank-p toolchain))
+                                                (equal "+" toolchain))
```

Comment:
Any reason why we are doing this check ?

Reply:
the + is in for just in case whatever it might be that no string was given to toolchain